source/brailleDisplayDrivers/handyTech.py

```diff
@@ -203,20 +204,32 @@ def display(self, cells):
 
 class AtcMixin(object):
 	"""Support for displays with Active Tactile Control (ATC)"""
+	supportedSettings=(
+		# Translators: Name of the ATC (active tactile control) setting in braille settings.
+		# Active tactile control should be left untranslated.
+		BooleanDriverSetting("atc",_("Enable &Active Tactile Control")),

```

Comment:
Makes sense. I commented it out. I'd like to keep the commented out version though, because it makes sense for documenting purposes. Feel free to suggest otherwise.